apps/style/applab/style.scss

```diff
@@ -459,11 +459,24 @@ button.arrow:focus {
   border-color: black;
   border-style: solid;
   border-width: 1px;
-  margin-top: 10px;
-  margin-bottom: 5px;
+  margin-top: 0px;
+  margin-bottom: 10px;
   font-size: 18px;
 }
 
+#designModeBox td {
+  vertical-align: top;
+}
+
+#divApplab * {
+  box-sizing: border-box;
```

Comment:
Changed this to:
#divApplab { padding: 2px; box-sizing: border-box; } // keeps dynamically-generated elements off edges of app
#divApplab input { box-sizing: border-box; }